runtime/altair/src/weights/pallet_block_rewards.rs

```diff
@@ -0,0 +1,58 @@
+
+//! Autogenerated weights for `pallet_block_rewards`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-03-10, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `Laptop.local`, CPU: `<UNKNOWN>`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("altair-local"), DB CACHE: 1024
+
+// Executed Command:
+// target/release/centrifuge-chain
+// benchmark
+// pallet
+// --chain=altair_local
+// --steps=50
+// --repeat=20
+// --pallet=pallet_block_rewards
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --output=/tmp/runtime/development/src/weights/pallet_block_rewards.rs
```

Comment:
It should be `altair` here, not sure if it was badly generated or it's till not generated.

Reply:
These weights are temporary anyways. Before the release, we will rerun all benchmarks.